### PR TITLE
Minor Improvements

### PR DESCRIPTION
**Pull request by arturobernalg**
* Add final
* Unused import
* Unnecessary semicolon
* Use Standard Charset object
* Compare
* Use primitive
* Replace block comment to text plain
* Redundant 'if' statement
* Fix @ mail info

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,49 +65,49 @@
     <developer>
       <name>James Strachan</name>
       <id>jstrachan</id>
-      <email>jstrachan -at- apache.org</email>
+      <email>jstrachan@apache.org</email>
```
Comment (garydgregory) [inline pom.xml:68]: -1 don't fiddle with these please.

File: commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/sftp/test/SftpProviderStreamProxyModeTestCase.java [hunk 2]
```diff
@@ -32,7 +32,7 @@
 
 import junit.framework.Test;
 
-public class SftpProviderStreamProxyModeTestCase extends AbstractSftpProviderTestCase {
+public class    SftpProviderStreamProxyModeTestCase extends AbstractSftpProviderTestCase {
```
Comment (garydgregory) [inline commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/sftp/test/SftpProviderStreamProxyModeTestCase.java:35]: Fix formatting.

File: commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/hdfs/test/HdfsFileProviderTest.java [hunk 3]
```diff
@@ -117,7 +117,7 @@ public static void setUmask(final Configuration conf2) {
             final BufferedReader bri = new BufferedReader(new InputStreamReader(p.getInputStream()));
             final String line = bri.readLine();
             p.waitFor();
-            final Short umask = Short.parseShort(line.trim(), 8);
+            final short umask = Short.parseShort(line.trim(), 8);
```
Comment (garydgregory) [inline commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/hdfs/test/HdfsFileProviderTest.java:120]: `Short.parseShort(line.trim(), 8)` -> `Short.parseShort(line.trim(), 8).shortValue()` to make unboxing explicit

**Review by garydgregory — CHANGES_REQUESTED** [review]

File: commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/sftp/test/SftpProviderStreamProxyModeTestCase.java (hunk 2 above)
Comment (arturobernalg) [reply commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/sftp/test/SftpProviderStreamProxyModeTestCase.java:35]: Done

File: pom.xml (hunk 1 above)
Comment (arturobernalg) [reply pom.xml:68]: :-( revert change

File: commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/hdfs/test/HdfsFileProviderTest.java (hunk 3 above)
Comment (arturobernalg) [reply commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/hdfs/test/HdfsFileProviderTest.java:120]: Don't fallow. Revert only this change

File: commons-vfs2/src/test/java/org/apache/commons/vfs2/RunTest.java [hunk 4]
```diff
@@ -40,7 +40,7 @@ public static void main(final String[] args) throws Exception {
         //props.setProperty("test.webdav.uri", "webdav://vfsusr:vfs%2f%25\\te:st@" + ip + "/vfstest");
         props.setProperty("test.sftp.uri", "sftp://vfsusr:vfs%2f%25\\te:st@" + ip + "/vfstest");
 
-        final Test tests[] = new Test[] {
+        final Test  tests[] = new Test[] {
```
Comment (garydgregory) [inline commons-vfs2/src/test/java/org/apache/commons/vfs2/RunTest.java:43]: Fix formatting.

Comment (arturobernalg) [reply commons-vfs2/src/test/java/org/apache/commons/vfs2/RunTest.java]: Fix it

**Pull request merged**